src/main/java/com/SAS/User/UserController.java

```diff
@@ -1,13 +1,14 @@
 package com.SAS.User;
 
-import java.util.HashSet;
+import com.SAS.crudoperations.CRUD;
 
 public class UserController {
 
     /**
      * singleton Privileges instance
      */
     private Privileges globalPrivileges;
+    private CRUD crud;
 
```

Comment:
update this because now Crud is static and you don't need instance of it to use it's functions

Reply:
I'll fix it.